confgenerator/confgenerator.go

```diff
@@ -424,126 +271,3 @@ func getUserAgent(prefix string, hostInfo *host.InfoStat) (string, error) {
 	}
 	return expandTemplate(userAgentTemplate, prefix, extraParams)
 }
-
-func generateFluentBitInputs(receivers map[string]LoggingReceiver, pipelines map[string]*LoggingPipeline) ([]fluentbit.Input, error) {
-	inputs := []fluentbit.Input{}
-	for _, pID := range sortedKeys(pipelines) {
```

Comment:
[Optional] There is now more unused code in `confgenerator/config.go`, e.g., type switch cases in `mapKeys`. Some structs in `confgenerator/fluentbit/conf.go` are probably unused as well…

Reply:
As I said to you in chat, "I want to start ripping out all the unused code, but I want to make sure the diffs are okay first before I lose the ability to switch back."

You've done your review, now give me a chance to actually delete things.